### PR TITLE
Read project VERSION from package.xml

### PR DESCRIPTION
**Pull request by henningkayser**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 3.5)
-project(geometric_shapes VERSION 1.0.0)
+project(geometric_shapes)
+
+# Set VERSION from package.xml
+find_package(ament_cmake_core REQUIRED)
```
Comment (rhaschke) [inline CMakeLists.txt:5]: You are going to `find_package(ament_cmake)` further below. Would that already include core?
Maybe move `ament_package_xml()` further down then?

**Review by rhaschke — COMMENTED** [review]
Thanks for looking this up! I couldn't find a hint to `ament_package_xml()` in a (quick) google search. Some minor remarks below.
Probably you should apply the same patch to the other repos (moveit2 and srdfdom) to get rid off the hard-coded version.

File: CMakeLists.txt (hunk 1 above)
Comment (henningkayser) [reply CMakeLists.txt:5]: `ament_cmake` works

**Comment by henningkayser** [comment]
> Probably you should apply the same patch to the other repos (moveit2 and srdfdom) to get rid off the hard-coded version.
 
Of course! I think this could even go into the [common macro](https://github.com/ros-planning/moveit2/pull/285) for moveit2.

File: CMakeLists.txt (hunk 1 above)
Comment (rhaschke) [reply CMakeLists.txt:5]: I adapted the suggestion appropriately in the main PR https://github.com/ros-planning/geometric_shapes/pull/162

**Pull request merged**